pkg/controller/master/image/vm_image_controller.go

```diff
@@ -272,6 +273,30 @@ func (h *vmImageHandler) createBackingImage(image *harvesterv1.VirtualMachineIma
 		bi.Spec.SourceParameters[lhmanager.DataSourceTypeExportFromVolumeParameterExportType] = lhmanager.DataSourceTypeExportFromVolumeParameterExportTypeRAW
 	case harvesterv1.VirtualMachineImageSourceTypeRestore:
 		bi.Spec.SourceParameters[lhv1beta2.DataSourceTypeRestoreParameterBackupURL] = image.Spec.URL
+	case harvesterv1.VirtualMachineImageSourceTypeClone:
+		bi.Spec.SourceParameters[lhv1beta2.DataSourceTypeCloneParameterEncryption] = string(image.Spec.SecurityParameters.CryptoOperation)
+
+		sourceImage, err := h.images.Get(image.Spec.SecurityParameters.SourceImageNamespace, image.Spec.SecurityParameters.SourceImageName, metav1.GetOptions{})
+		if err != nil {
+			return fmt.Errorf("failed to get source vmimage %s/%s, error: %s", image.Spec.SecurityParameters.SourceImageName, image.Spec.SecurityParameters.SourceImageNamespace, err.Error())
```

Comment:
```suggestion
			return fmt.Errorf("failed to get source vmimage %s/%s, error: %s", image.Spec.SecurityParameters.SourceImageNamespace, image.Spec.SecurityParameters.SourceImageName, err.Error())
```

Nit: just to be consistent with other log messages.